decidim-comments/app/packs/src/decidim/comments/comments.component.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
Do we need this?

If I'm calculating correctly, theres 8 lines added, 8 lines removed.